docs/source/python_spec.rst

```diff
@@ -134,6 +134,36 @@ C API which is called either when the refcount on the capsule named
 Note: the capsule names ``"dltensor"`` and ``"used_dltensor"`` must be
 statically allocated.
 
+The ``DLManagedTensor`` deleter must ensure that sharing beyond Python
+boundaries is possible, this means that the GIL must be acquired explicitly.
+In Python, the deleter usually needs to ``Py_DECREF()`` the original owner
+and free the ``DLManagedTensor`` allocation.
+For example, NumPy uses the following code to ensure sharing with arbitrary
+non-Python code is safe:
+
+.. code-block:: C
+   static void array_dlpack_deleter(DLManagedTensor *self)
```

Comment:
You might want to add a newline here to appease the reST parser?
```suggestion

   static void array_dlpack_deleter(DLManagedTensor *self)
```

Reply:
See CI error:
https://github.com/dmlc/dlpack/actions/runs/8387657417/job/22970268932?pr=140#step:5:156